apps/meteor/client/hooks/useFontStylePreference.tsx

```diff
@@ -0,0 +1,16 @@
+import { useUserPreference } from '@rocket.chat/ui-contexts';
+import { useLayoutEffect } from 'react';
+
+import { useCreateFontStyleElement } from '../views/account/accessibility/hooks/useCreateFontStyleElement';
+
+export const useFontStylePreference = () => {
+	const fontSize = useUserPreference('fontSize', '100%');
+	const createFontStyleElement = useCreateFontStyleElement();
+
+	useLayoutEffect(() => {
+		if (!fontSize) {
+			return;
+		}
+		createFontStyleElement(fontSize);
```

Comment:
What happens if the preference changes?

Reply:
this is handled on the Accessibility page, it overwrites the css value of `font-size`.
it's not creating duplicates, but I could add the condition to query the element by id `rcx-font-size` before creating it:
```
const styleElement = document.getElementById('rcx-font-size') || createStyleElement('rcx-font-size');
```
wdyt?